changelog/upgrade-notes.md

```diff
@@ -6,6 +6,11 @@
 
 The experimental `setHue`, `setContrast`, `setBrightness`, `setSaturation`, and the corresponding getter methods have been removed.  These properties only worked with the WebGL renderer.  If are interested in applying color transforms, look for the `postcompose` event in the API docs.  In addition, the `ol.source.Raster` source provides a way to create new raster data based on arbitrary transforms run on any number of input sources.
 
+#### Removal of `ol.ENABLE_NAMED_COLORS`
+
+This option existed to support the use of Google Closure library named colors and was removed as a part of OpenLayers moving away from Google Closure library. Note that this do not affect the use of CSS named colors.
```

Comment:
I agree it would be nice to be more explicit about what this change means.  I would remove all reference to the Google Closure Library, since I don't think that is relevant.  It would be more relevant for people to know if they need to change their application code.